packages/dashboard/src/components/Card/Chart/Chart.types.ts

```diff
@@ -34,6 +34,13 @@ export enum ChartHeight {
   tall
 }
 
+export enum TimeRange {
```

Comment:
Nit: prefer union string types (`'7Days' | '30Days' | '90Days' | '180Days'`) rather than an enum.

Reply:
as discussed updated to string union type